vela-templates/definitions/internal/trait/startup-probe.cue

```diff
@@ -0,0 +1,109 @@
+"startup-probe": {
+	type: "trait"
+	annotations: {}
+	description: "Add startup probe hooks for every container of K8s pod for your workload which follows the pod spec in path 'spec.template'."
+	attributes: {
+		podDisruptive: true
+		appliesToWorkloads: ["deployments.apps", "statefulsets.apps", "daemonsets.apps", "jobs.batch"]
+	}
+}
+template: {
+	patch: spec: template: spec: containers: [...{
```

Comment:
Here we usually use //+patchKey=name, or use [{}, ...]. The [...{}] for patch means the startupProbe will be patched to all containers.